install-monorepo.sh

```diff
@@ -29,63 +29,71 @@ fi
 
 # Check of Homebrew and nvm are installed
 echo "Checking if Homebrew is installed..."
-if [[ -z "$(command -v brew)" ]]; then
+if ! command -v brew &>/dev/null; then
     echo "Installing Homebrew"
     /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-	if [[ ON_LINUX ]]; then # Add homebrew to PATH
-		echo 'eval "$(/home/ubuntu/.linuxbrew/bin/brew shellenv)"' >> /home/ubuntu/.profile
-		eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
-		PATH=/home/linuxbrew/.linuxbrew/bin:$PATH
-		hash -r 
+	if [[ -n "$ON_LINUX" ]]; then # Add homebrew to PATH
+	# Reset the terminal so it picks up the changes.
```

Comment:
No resetting going on here. Just adding things to the profile and environment.

Reply:
Yeah this was a bad commit where I made a mess trying to sort out the above that is now solved with || true. Testing a few things and I'll fix it up again.